doc/en/user/source/services/wms/configuration.rst

```diff
@@ -64,3 +64,13 @@ The following table shows examples of reasonable values for the request limits:
      - 100
      - Encountering 100 errors is probably the result of a request trying to reproject a big data set into a projection that is not appropriate for the output extent, resulting in many reprojection failures.
 
+
+GetCapabilities Settings
+------------------------
+.. list-table::
+   :widths: 20 70
+
+   * - **Option**
+     - **Description**
+   * - Include Default LayerGroup Style In GetCapabilities
+     - Enable/disable the encoding of the default layerGroup style in GetCapabilties responses. By default is set to enabled.
```

Comment:
Does that happen for all types of layer groups? We have some indication that encoding a defatul style for tree based groups (e.g. with visible layers nested inside) are going to cause troubles due to the style being inherited, but not really available for the single layers.

Reply:
fixed now the option allows to disable the default style for tree types only

Reply:
fixed